### DISCUSSION TITLE
How to insert an object into the list at the starting

### DISCUSSION THREAD

**Discussion by traghu84** [mikefarah/yq#1544] [Q&A]
How do i insert an object into the starting or ending of a list of objects.
Ex:
Below is my yaml
microservices:
  - app: login
    port: 443
  - app: order
    port: 8080

And i want to insert a new record as below at the starting of the list.
microservices:
  - app: deliver
    port: 8081
  - app: login
    port: 443
  - app: order
    port: 8080

Is there an easy way to do this. Can someone please help me with your suggestions.

**Comment by mikefarah** [comment]
Have a look here: https://mikefarah.gitbook.io/yq/operators/add#append-to-existing-array, and directly underneath that is how to prepend :)
